app/views/jobs/group.html.erb

```diff
@@ -19,6 +20,9 @@
     <p class="not_success"><b><%= @group_jobs.first["name"] %></b> is overdue.</p>
   <% end %>
 
+  <div class="pagination justify-content-end">
```

Comment:
The pagination links aren't styled in firefox, likely because this div is not valid within a table.  If you move the pagination links outside the table (after it is probably best since users won't really care about them until they've scrolled all the way down anyway), it might work.